inst/shiny/server.R

```diff
@@ -39,14 +39,14 @@ shinyServer(function(input, output, session) {
     
     updateCheckboxGroupInput(session, "area", 
                              choices = choices, 
-                             selected = choices)
+                             selected = choices[1])
```

Comment:
I found it cumbersome to uncheck a ton of stuff so switched this to only select one initially... maybe there's a check all uncheck all option? I couldn't find it.